pkg/core/linter.go

```
@@ -84,6 +84,16 @@ type LinterOptions struct {
 	// GitHubToken は GitHub API を呼ぶルールに提示するトークン。
 	// 空文字なら未認証 60 req/h 制限が適用される。
 	GitHubToken string
+	// ReportFilePaths limits rendered findings to the listed repository-relative
+	// paths while still analysing every file passed to LintFiles. This is used
```

golangci-lint (2.12.2)

[error] `analysing` is a misspelling of `analyzing`

(misspell)